apps/scope/pkg/apis/scope/v0alpha1/types.go

```diff
@@ -0,0 +1,233 @@
+package v0alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+/*
+Please keep pkg/promlib/models/query.go and pkg/promlib/models/scope.go in sync
+with this file until this package is out of the grafana/grafana module.
+*/
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type Scope struct {
+	metav1.TypeMeta   `json:",inline"`
```

Comment:
since this isn't built with the app SDK and the rest of the scopes code lives elsewhere, what do you think about moving this to `pkg/apis/` and making `pkg/apis` or just `pkg/apis/scope` a sub module? cc @radiohead @IfSentient

Reply:
I don't have strong preferences right now, but ideally I'd rather have all APIs / apps live in `apps/$APP_NAME`, regardless of whether they are built using SDK or not.

Over time we _do_ want to make sure that all apps are built using the SDK but putting all of them in one place is a good start IMO.

CC @stephaniehingtgen @ryan-grafana

Reply:
We can build _some_ of the types in CUE, so I can refactor it that way too!